tests/webhook_test.go

```diff
@@ -164,7 +172,7 @@ var _ = Describe("Validation webhook", func() {
 					foundSsp = getSsp()
 					foundSsp.Spec.TemplateValidator.Placement = &placementAPIValidationValidPlacement
 					return apiClient.Update(ctx, foundSsp, client.DryRunAll)
-				}, twentySecondTimeout, time.Second).ShouldNot(HaveOccurred(), "failed to update SSP CR with valid template-validator placement fields")
```

Comment:
Why do you remove this variable? Lets define it in env package

Reply:
I thought the variable was not important. It has no other meaning than 20 seconds, so it can be inlined.